lua/nvim-web-devicons/icons-default.lua

```diff
@@ -2084,6 +2810,12 @@ local icons_by_file_extension = {
     cterm_color = "166",
     name = "Xml",
   },
+  ["xpi"] = {
```

Comment:
Is this mozilla specific? `nf-dev-mozilla` if it is.

Reply:
I think it is